monitoring/tracer/run_poll_locally.sh

```diff
@@ -7,6 +7,8 @@ LOGS='--output-folder=/logs'
 KML_SERVER='--kml-server=https://example.com/kmlgeneration'
 KML_FOLDER='--kml-folder=test/localmock'
 MONITOR='--rid-isa-poll-interval=15 --scd-operation-poll-interval=15 --scd-constraint-poll-interval=15'
+# TODO: Fix unused
+# shellcheck disable=SC2034
```

Comment:
This is a polling routine and doesn't host an endpoint so `PORT=` is definitely not needed; that line can be deleted instead.